### ISSUE TITLE
build error:  TypeError: o.jsxDEV is not a function

### ISSUE THREAD

**Issue by jackChauT** [Wolox/react-chat-widget#267]
Hi, 
I downloaded the project and run `npm install`, then `npm run build`.
Build was successful. Then I run `npm install path_to_my_react-chat-widget`. 

so my dependencies is
`"dependencies": {..., 
    "react-chat-widget": "file:third_party/react-chat-widget", ...}`

I use `import { Widget } from 'react-chat-widget';` in my code but I got the error `o.jsxDEV is not a function`

How do i fix it? Thank you so much!

<img width="1440" alt="Screenshot 2021-07-23 at 1 34 45 PM" src="https://user-images.githubusercontent.com/51106573/126741185-075b2a97-b4b7-459f-8f30-2a575fa9c557.png">
<img width="1440" alt="Screenshot 2021-07-23 at 1 42 23 PM" src="https://user-images.githubusercontent.com/51106573/126741589-7727fec0-193a-4209-80c5-f184174f3b34.png">

**Comment by bharath-naik** [comment]
Hello mate,
I am getting the same error. :/

**Comment by bharath-naik** [comment]
seems to be some babel versioning issue :(

**Comment by hichemfantar** [comment]
Bump please fix this

**Comment by vinodveloo** [comment]
Hi same error :/

**Comment by yashrajthakor** [comment]
same issue faced

**Comment by yashrajthakor** [comment]
npm i  react-chat-widget@3.0.5    => it's worked if you downgrade the version.

**Comment by sh1okoh** [comment]
Hi same error, thx

**Comment by smrutikant** [comment]
Mate, any fixes?

**Comment by chiragj97** [comment]
Only solution that worked for me - npm install react-chat-widget@3.0.5

**Comment by DragoshDX** [comment]
Hello, as of 28 September 2021, that issue is still happening and downgrading to 3.0.5 seems to work indeed. 

I can provide more details of my setup for the maintenance team (let me know if I can help)

**Comment by chiragj97** [comment]
Hello,
Yes, 3.0.5 works fine but I'm not able to use the profileClientAvatar in
this version. Also is there a prop to select file from the system to upload
an image as a chat message ??

Thanks,
Chirag Jain

**Comment by DragoshDX** [comment]
Correct, profileClientAvatar and even emojis, seem to have been added after 3.0.5

**Comment by chiragj97** [comment]
Got it. Any idea for how I can choose a file from my system from the
widget. I mean I wanted an input tag with type="file". Is that possible?

**Comment by DragoshDX** [comment]
What I did was hide the form from the implementation and stick a different form underneath with an upload button and a send button. The send button just uses the imperative API to push messages to the chat container.

**Comment by krajcikondra** [comment]
Problem is in commit 08e421e2d2d5a446a501ead3e80db4cc3249bfb8 but I am not able to fix it

**Comment by Lightfooted** [comment]
I have this same problem. Downgrading to 3.0.5 and it works. The newest version brings up either a blank page or the error at random of jsxDEV is not a function.

**Comment by aravindd7** [comment]
> > Hello, as of 28 September 2021, that issue is still happening and downgrading to 3.0.5 seems to work indeed.
> > I can provide more details of my setup for the maintenance team (let me know if I can help)
> 
> I have "react": "^17.0.1", "react-chat-widget": "^3.0.5",
> 
> I still get TypeError: r.jsxDEV is not a function

Even i have same issue. downgraded version still error persists

**Comment by budarin** [comment]
has anyone figured out what the problem is?

it's the first time I've encountered her in so many years and I don't understand what the problem is!

I am assembling the project and configuration in production mode, but the cat is apparently for development

**Comment by clicktodev** [comment]
@budarin I don't recommend this package, It's extremely outdated. Probably better to build it yourself, It's not complicated at all.
